lib/msf/ui/console/command_dispatcher/db/klist.rb

```diff
@@ -118,4 +134,66 @@ def cmd_klist(*args)
   def kerberos_ticket_storage
     @kerberos_ticket_storage ||= Msf::Exploit::Remote::Kerberos::Ticket::Storage::ReadWrite.new(framework: framework)
   end
+
+  private
+
+  def ticket_status(ticket)
+    if ticket.expired?
+      '>>expired<<'
+    elsif ticket.status.blank?
+      'active'
+    else
+      ticket.status
+    end
+  end
+
+  # Sets the status of the tickets
+  #
+  # @param [Symbol] mode The status (:activate or :deactivate) to apply to the ticket(s)
+  # @param [Array<StoredTicket>] tickets The tickets which statuses are to be updated
+  # @return [Integer] Number of tickets updated
+  def set_activation_status(mode, tickets)
```

Comment:
TBH I'm not entirely sure what happened here, takes in a list of tickets turns it into a list of IDs and then fetches them each one by one from the DB in `(de)activate_ccache` I think it got lost in all the changes
anyway, not sure what to do about it either pass the tickets the whole way through or only accept and array of IDs and fetch them later, but definitely not this halfway merge between the two options